ios/sdk/WeexSDK/Sources/Component/WXCellComponent.m

```diff
@@ -84,8 +95,8 @@ - (void)updateAttributes:(NSDictionary *)attributes
 
 - (void)_moveToSupercomponent:(WXComponent *)newSupercomponent atIndex:(NSUInteger)index
 {
-    if (self.list == newSupercomponent) {
-        [self.list cell:self didMoveToIndex:index];
+    if (self.delegate == newSupercomponent) {
```

Comment:
This method is required to be implemented in protocol, and the protocol will not be exported to public,  maybe it's just controllable.